cmd/snap/wait.go

```diff
@@ -117,7 +120,7 @@ func (wmx waitMixin) wait(id string) (*client.Change, error) {
 
 		for _, t := range chg.Tasks {
 			switch {
-			case t.Status != "Doing":
+			case t.Status != "Doing" && t.Status != "Wait":
```

Comment:
I don't understand this change, as Wait is similar to Done and Hold, not Doing

Reply:
This was added so that the next case in the switch statement can run, i.e. so that the log.Noticef() with the task log is displayed. I realize this is a bit confusing, I need think I guess if there is a more natural way to express this. The task.Log() of a Wait task contains the piece that I want to display to the user.